cdap-app-templates/cdap-etl/cdap-etl-api/src/main/java/io/cdap/cdap/etl/api/Joiner.java

```diff
@@ -17,26 +17,48 @@
 package io.cdap.cdap.etl.api;
 
 import io.cdap.cdap.api.annotation.Beta;
+import org.apache.commons.lang.NotImplementedException;
```

Comment:
avoid using classes from other libraries in the api modules. You can throw a UnsupportedOperationException instead.

Reply:
done